egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -147,7 +149,7 @@ if [ $# != 4 ]; then
   
   exit 1;
 fi
-
+set -x
```

Comment:
This is for debugging

"set -x: set this flag to trace the execution of all commands. This will make the interpreter print each command right before it is executed, so it will aid you in knowing what is happening at any point in time."

This will be removed when the recipe is finalized.